### PR TITLE
Add version parameters to KCP CLI on Kyma upgrade

### PR DESCRIPTION
**Pull request by fhivemind**
**Description**

Allows KCP to specify a custom version on Kyma upgrade.

### CODE REVIEW

File: tools/cli/pkg/command/upgrade_kyma.go [hunk 1]
```diff
@@ -24,11 +24,13 @@ func NewUpgradeKymaCmd() *cobra.Command {
...
+The version is specified via the --version (or -v) options. If empty, the version will be configured by Kyma Environment Broker (KEB).
+Additional Kyma configurations to use for the upgrade are taken from Kyma Control Plane during the processing of the orchestration.`,
 		Example: `  kcp upgrade kyma --target all --schedule maintenancewindow     Upgrade Kyma on all Runtimes in their next respective maintenance window hours.
   kcp upgrade kyma --target "account=CA.*"                       Upgrade Kyma on Runtimes of all global accounts starting with CA.
   kcp upgrade kyma --target all --target-exclude "account=CA.*"  Upgrade Kyma on Runtimes of all global accounts not starting with CA.
-  kcp upgrade kyma --target "region=europe|eu|uk"                Upgrade Kyma on Runtimes whose region belongs to Europe.`,
+  kcp upgrade kyma --target "region=europe|eu|uk"                Upgrade Kyma on Runtimes whose region belongs to Europe.
+  kcp upgrade kyma --target all --version "master-00e83e99"      Upgrade Kyma on Runtimes of all global accounts to custom Kyma version master-00e83e99.`,
```
Comment (ebensom) [inline tools/cli/pkg/command/upgrade_kyma.go:33]: Run `make docs` to update the markdown docs.

**Review by ebensom — DISMISSED** [review]

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ Upgrades or reconfigures Kyma on one or more Kyma Runtimes.
 Upgrades or reconfigures Kyma on targets of Runtimes.
 The upgrade is performed by Kyma Control Plane (KCP) within a new orchestration asynchronously. The ID of the orchestration is returned by the command upon success.
 The targets of Runtimes are specified via the `--target` and `--target-exclude` options. At least one `--target` must be specified.
-The Kyma version and configurations to use for the upgrade are taken from Kyma Control Plane during the processing of the orchestration.
+The version is specified via the `--version` (or -v) options. If empty, the version will be configured by Kyma Environment Broker (KEB).
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:10]:
```suggestion
The version is specified using the `--version` (or -v) option. If not specified, the version is configured by Kyma Environment Broker (KEB).
```

File: docs/cli/commands/kcp_upgrade_kyma.md [hunk 3]
```diff
@@ -20,6 +21,7 @@ kcp upgrade kyma --target {TARGET SPEC} ... [--target-exclude {TARGET SPEC} ...]
   kcp upgrade kyma --target "account=CA.*"                       Upgrade Kyma on Runtimes of all global accounts starting with CA.
   kcp upgrade kyma --target all --target-exclude "account=CA.*"  Upgrade Kyma on Runtimes of all global accounts not starting with CA.
   kcp upgrade kyma --target "region=europe|eu|uk"                Upgrade Kyma on Runtimes whose region belongs to Europe.
+  kcp upgrade kyma --target all --version "master-00e83e99"      Upgrade Kyma on Runtimes of all global accounts to custom Kyma version master-00e83e99.
```
Comment (klaudiagrz) [inline docs/cli/commands/kcp_upgrade_kyma.md:24]:
```suggestion
  kcp upgrade kyma --target all --version "master-00e83e99"      Upgrade Kyma on Runtimes of all global accounts to the custom Kyma version (master-00e83e99).
```

**Review by klaudiagrz — APPROVED** [review]

**Review by ebensom — APPROVED** [review]

**Pull request merged**